packages/extension-chakra-storefront/src/hooks/use-datacloud.test.js

```diff
@@ -63,14 +63,6 @@ jest.mock('./use-current-customer', () => ({
         }
     })
 }))
-// jest.mock('js-cookie', () => {
```

Comment:
If we revert by un-commenting this `jest.mock`, you'll see the main reason why data cloud tests were failing: for some reason, jest had trouble mocking the js-cookie lib and ran into out-of-memory error.

So let's avoid mocking it.